### DISCUSSION TITLE
What was the name of that other guy with the good implementation of a computer emulator in the browser?

### DISCUSSION THREAD

**Discussion by 621625** [copy/v86#765] [General]
Hello,

I'm BM. A very nice and polite guy.

What was the name of that other guy with the good current implementation of a computer emulator in the browser? I forget the name of the project. American or Canadian guy. Sharp implementation. The emulator loads quickly and can run a lot of reasonably contemporary applications. He also made some YouTube videos about it. Anyone remembers?

**Comment by MentalGear** [comment]
pcjs maybe.

**Reply by spetterman66** [reply]
jslinux? nepx/halfix (github)?
